src/ResourceManager/KeyVault/Commands.KeyVault.Test/Scripts/PSHCommon/Common.ps1

```diff
@@ -298,7 +298,7 @@ function Wait-Seconds
 {
     param([int] $timeout)
     
-    [Microsoft.WindowsAzure.Testing.TestUtilities]::Wait($timeout * 1000)
+    Sleep -s $timeout
```

Comment:
@karlaugsten This still causes management overhead, beceause whenever we audit the cmdlet tests, we have to make an exception for this one.  Please use the existing library function (which has exactly the functionality you need for live tests) to wait inside your script.